projects/wave-core-new/src/lib/time/time-slider/time-slider.component.scss

```diff
@@ -0,0 +1,5 @@
+.visualization {
+    width: 100%;
+    height: 150px;
```

Comment:
I guess it is not good to fix this here. We need different heights for different positions.